src/Components/HistoricalDataChart/HistoricalDataChart.scss

```diff
@@ -1,36 +1,17 @@
-.chartContainer{
-   display: flex;
-   flex-direction: column;
-   align-items: flex-start;
-   position: absolute;
-   top: 0;
-   right: 0;
-   margin-right: 8%;
-}
-
-.dropdownContainer {
-   margin-right: 200px;
-   align-self: flex-end;
-}
-
-.dateDropdown{
-   padding-left: 8px;
-}
+.chartDateFilter {
+   .dateDropdown{
+      padding-left: 8px;
+      float:right;
+   }
 
-.loadingErrorContainer {
-   width: 756px;
-   height: 275px;
+   :first-child {
+      vertical-align: bottom !important;
+   }
 }
-
-// Media Query
-@media (max-width: 1620px ){
-   .chartContainer {
-      position: static; 
-   } 
-
-    .dropdownContainer {
-      margin-left: 56px;
-      align-self: center;
-      margin-right: 0px;
+.historical-chart-panel {
+   @media screen and (min-width: 768px) {
+      .pf-c-chart {
+         top: -65px;
+      }
```

Comment:
@kgaikwad I am just curious to know why  we had to add the top value, can you please share the reason?

Reply:
![209525801-ae5ad81a-5012-40d3-85c9-e2b6ed214ffd](https://user-images.githubusercontent.com/6470528/209962046-c98911b4-4f21-446f-96e9-2f68a56580fc.png)
^^ to fix charts & dropdown's alignment as per mockup